modules/ztp-image-based-upgrade-installing-oadp.adoc

```diff
@@ -248,11 +250,12 @@ spec:
                 key: cloud
                 name: cloud-credentials
               objectStorage:
-                bucket: <your_bucket_name> <3>
-                prefix: <cluster_name> <3>
+                bucket: <your_bucket_name> <4>
+                prefix: <cluster_name> <4>
     - fileName: OadpBackupStorageLocationStatus.yaml
       policyName: "config-policy"
 ----
 <1> Specify your credentials for your S3 storage backend.
-<2> Specify the URL for your S3-compatible bucket.
-<3> The `bucket` defines the bucket name that is created in S3 backend. The `prefix` defines the name of the subdirectory that will be automatically created in the `bucket`. The combination of `bucket` and `prefix` must be unique for each target cluster to avoid interference between them. To ensure a unique storage directory for each target cluster, you can use the {rh-rhacm} hub template function, for example, `prefix: {{hub .ManagedClusterName hub}}`.
\ No newline at end of file
+<2> If there are more than one `backupLocation` defined in the `OadpDataProtectionApplication` CR, each `backupLocation` must have a corresponding `OadpBackupStorageLocation` CR added for status tracking. Ensure that the name of each additional `OadpBackupStorageLocation` CR is overridden with the correct index as described in the example `OadpBackupStorageLocationStatus.yaml` file.
```

Comment:
```suggestion
<2> If more than one location is specified under `backupLocations` in the `OadpDataProtectionApplication` CR, ensure that each location has a corresponding `OadpBackupStorageLocation` CR added for status tracking. Ensure that the name of each additional `OadpBackupStorageLocation` CR is overridden with the correct index as described in the example `OadpBackupStorageLocationStatus.yaml` file.
```

Reply:
To avoid using directional words, I'm rephrasing this as follows:
> If more than one `backupLocations` entries are defined in the `OadpDataProtectionApplication` CR, ...

Reply:
I might have gone with _If more than one `backupLocations` entry is defined_. Not worth holding up the merge.